src/stores/filingHistoryListStore.ts

```diff
@@ -29,7 +29,7 @@ export const useFilingHistoryListStore = defineStore('filingHistoryList', {
       // return only the valid filings
       return state.filings.filter(filing => {
         // safety check for required fields
-        if (!filing.name || !filing.displayName || !filing.effectiveDate || !filing.submittedDate || !filing.status) {
+        if (!filing.name || !filing.displayName || !filing.effectiveDate || !filing.submittedDate || !filing.status || !filing.displayLedger) {
```

Comment:
This will work but I think you should add it as a second if statement so that no console.log is displayed (line 34) if we are not to display this ledger item.